rhino/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+Rhino supports two editors: IronPython and CPython. The following tutorials explain how to install and use compas on both.
+
+# Installation - Editor: IronPython
+
+- **Step 1:** Install Anaconda: https://www.anaconda.com/products/distribution
+
+- **Step 2:** Anaconda Prompt commands:
+
+    ```bash
+
+    conda create -n compas-dev -c conda-forge compas
+    conda activate compas-dev
+    python -m compas_rhino.install -v7.0
```

Comment:
is there a missing space between `v` and `7.0`?

Reply:
Both space and no space works, I will add space:

<img width="752" alt="image" src="https://github.com/compas-dev/tutorials/assets/18013985/3928b961-bb20-4afb-8279-d37bee9af385">